Lib.AspNetCore.ServerSentEvents/ServerSentEventsMiddleware.cs

```diff
@@ -114,6 +114,8 @@ public async Task Invoke(HttpContext context, IPolicyEvaluator policyEvaluator)
             }
         }
 
+        private static readonly string[] AcceptedHeaders = new[] { "*/*", "text/*", Constants.SSE_CONTENT_TYPE };
```

Comment:
I would argue that iterating a small array is faster than using hashset. But I don't mind changing it if you prefer the more scalable option.

Reply:
Please do :)

Reply:
We can also be really professional about it and benchmark it. Currently there are only benchmarks for `ServerSentEventsService` so that would mean setting up a benchmarks for `ServerSentEventsMiddleware` from scratch.

Reply:
I did a benchmark on hashset.contains and array.contains with a bounded dataset of 3 entries. And hashset seems superior. Especially when the hashset/array does not contain the string that is being looked up. I wasn't sure how to test this in a good way in your application, I'm not super well-versed in benchmarkdotnet. 

Regardless I have substituted the array for a hashset .

Reply:
Thank you :)